src/config.ts

```diff
@@ -1754,6 +1753,15 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 				coinGeckoId: 'pool:raw',
 				coinImageUrl: window.location.origin + '/public/assets/tokens/raw.png',
 			},
+			{
+				type: 'cw20',
+				contractAddress: 'juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3',
+				coinDenom: 'JOE',
+				coinMinimalDenom: 'cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3:JOE',
+				coinDecimals: 6,
+				coinGeckoId: 'pool:joe',
```

Comment:
Is there a liquidity pool to define what 'pool:joe' is? Strongly preferred, but can be remain undefined for a little while.

Reply:
@JeremyParish69 not yet